sandbox/apparmor/profile.go

```diff
@@ -98,10 +98,25 @@ func LoadProfiles(fnames []string, cacheDir string, flags AaParserFlags) error {
 	if !osutil.GetenvBool("SNAPD_DEBUG") {
 		args = append(args, "--quiet")
 	}
-	args = append(args, fnames...)
 
-	output, err := exec.Command("apparmor_parser", args...).CombinedOutput()
-	if err != nil {
+	parser, requiredArgs, internal, err := FindAppArmorParser()
+	if err != nil || !internal {
```

Comment:
Since this fallback is only required when doing unit testing to mock the parser command, I think perhaps I'll remove it entirely and update the unit tests to make sure they all MockParserSearchPath() as expected and then hopefully it "just works"

Reply:
See fd767ee5af